urllib3/util/queue.py

```diff
@@ -0,0 +1,21 @@
+import collections
+from ..packages import six
+from ..packages.six.moves import queue
+
+if six.PY2:
+    # Queue is imported for side effects on MS Windows
```

Comment:
No kidding? Got a link that explains what these side-effects are and why they're useful?

Reply:
Let me find the commit where that was added in the blame.

Reply:
Found the reason and PR where this was added: #229 https://github.com/urllib3/urllib3/commit/18e6e46ec7da085269ad4c7fcf01b13a7c572484
I'll add a link to it so no one has to go mucking through the enormous history log like I just did again. :)